src/templates/about/index.hbs

```diff
@@ -47,9 +48,9 @@
         </div>
         <div class="text">
           <h2>Scott Noell</h2>
-          <p>With 30 years' experience in the financial planning industry, Scott Noell is a partner and key driver of the FinSec Partners business. Scott built his impressive career- foundations specifically within insurance and superannuation before applying
+          <p>With 30 years experience in the financial planning industry, Scott Noell is a partner and key driver of the FinSec Partners business. Scott built his impressive career foundations specifically within insurance and superannuation before applying
             himself to more holistic financial planning practices.</p>
-          <p>Over the last 15 years Scott has actively developed the pension transfer business within FinSec, originally in a working relationship with Pension Transfers Direct and now via FinSec PTX.</p>
+          <p>Over the last 15 years Scott has actively developed the pension transfer business within FinSec, originally in a working relationship with Pension Transfers Direct and now via FinSec’s specialist transfer service FinSec PTX.</p>
```

Comment:
Ok important to check that you don't copy in none-UTF-8 characters such as the curly `'`. In the <p> above please replace `FinSec’s` with `FinSec's`.